amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -501,4 +513,99 @@ type ColonyAction @model {
   toDomain: Domain @hasOne(fields: ["toDomainId"])
   fundamentalChainId: Int
   newColonyVersion: Int
+  roles: ColonyActionRoles
```

Comment:
Nitpick: couldn't this point to the ColonyHistoricRoles entry created at the same time? They contain the same info, no?

Reply:
So these are different from the Historic Roles, this just has the exact roles that changed vs. historic role which have the exact snapshot at the time of them being set.

Eg: 
Action Role:
```
role_0: true
role_1: false
```
vs
```
role_0: true
role_1: false
role_2: null
role_3: null
role_5: null
role_6: null
```